### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by lemur117** [c-icap/c-icap-server#17]
The installation instructions seem to be missing some steps before the "./configure" step, and I can't figure out how to get the VERSION.m4 file needed by autoconf:

```
~/git/c-icap-server$ autoconf 
/usr/bin/m4:configure.ac:4: cannot open `VERSION.m4': No such file or directory
autom4te: /usr/bin/m4 failed with exit status: 1
```

**Comment by nabbi** [comment]
Agreed, I too found this step confusing. Take a look at [RECONF](https://github.com/c-icap/c-icap-server/blob/6812dfc896f4e032d254b01476cee9d9d9fc0f96/RECONF) for preparing the build environment.

This Gentoo [ebuild](https://github.com/nabbi/oubliette-overlay/blob/master/net-proxy/c-icap/c-icap-9999.ebuild) performs a similar command within src_prepare and might provide additional insights for you.
